front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarPhases.tsx

```diff
@@ -33,27 +35,40 @@ const FilterSidebarPhases = ({
     return selectedPhase === id;
   };
 
+  const name = 'phase';
+
   return (
-    <Menu
-      secondary={true}
-      vertical={true}
-      fluid={true}
-      className="intercom-admin-input-manager-phases"
-    >
-      <Menu.Item onClick={clearFilter} active={!selectedPhase}>
-        <FormattedMessage {...messages.allPhases} />
-      </Menu.Item>
-      <Divider />
-      {phases.map((phase, index) => (
+    <Box display="flex" flexDirection="column">
+      {/* FilterRadioButton is also used inside FilterSidebarPhasesItem */}
+      <FilterRadioButton
+        id="all-phases"
+        isSelected={!selectedPhase}
+        onChange={clearFilter}
+        labelContent={
+          <LabelContentWrapper>
+            <FormattedMessage {...messages.allPhases} />
+          </LabelContentWrapper>
+        }
+        name={name}
+      />
+      <Box
+        width="100%"
+        as="hr"
+        border={`1px solid ${colors.background}`}
+        mb="8px"
+      />
```

Comment:
[Ticket](https://www.notion.so/govocal/Create-reusable-Divider-1629663b7b2680fbb927f1ec1f30871d?pvs=4) to create a reusable version.